### ISSUE TITLE
Machine Learning(Dropout layers)

### ISSUE THREAD

**Issue by Enrique2007s** [Enrique2007s/skin-cancer-detector#12]
As a data scientist, I want to use dropout layers so that I prevent overfitting.

Define neural network architecture with dropout layers.
